modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -4,21 +4,21 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) is composed of a Salesforce organization with Community Cloud and a managed package that is installed in this organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization.
+Anypoint API Community Manager is composed of a Salesforce organization with Community Cloud and a managed package that is installed in this organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization.
 
-The ACM package is installed in an *independent* Salesforce organization from any other Salesforce organizations your company may already have, so that the API community data is isolated from your company operational data, such as ERP or CRM data.
+The API Community Manager package is installed in an *independent* Salesforce organization from any other Salesforce organizations your company may already have, so that the API community data is isolated from your company operational data, such as ERP or CRM data.
```

Comment:
Do not use bold for emphasis. Try to provide emphasis by context. If necessary, use _italic_:
```suggestion
The API Community Manager package is installed in a Salesforce organization that is _independent_ of any other Salesforce organizations that your company might already have, so that the API community data is isolated from your company operational data, such as ERP or CRM data.
```